src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,17 @@
 
 public class MainApp {
```

Comment:
The MainApp class should not have unnecessary empty lines after the class declaration. Please remove the empty line to adhere to the checklist guidelines.